solution.txt

```diff
@@ -0,0 +1,252 @@
+*/
+
+// Complete Exercise 10 below...
+
+
+
+console.log('Exercise 10 Result:');
+var hasSoup = true;
+if (foods.indexOf('soup') !== -1) {
+  hasSoup = true
+} else {
+  hasSoup = false
+}
+console.log(hasSoup)
+
+
+/*
+Exercise 11:
+  - Use the forEach method to iterate through the provided nums array and add each odd number to a new array named odds.
+  - Hint: Initialize the odds variable to an empty array before the iteration.
+*/
+
+
+
+console.log('Exercise 11 Result:');
+const nums = [100, 5, 23, 15, 21, 72, 9, 45, 66, 7, 81, 90];
+var odds = []
+nums.forEach(function(item){
+    if (item % 2 === 1) {
+    odds.push(item);
+    }
+});
+console.log(odds)
+
+/*
+Exercise 12:
+  - Use the forEach method to iterate through the same nums array and add the number to arrays named fizz, buzz and/or fizzbuzz based upon the following:
+  	- Add to the fizz array if the number is evenly divisible by 3.
+  	- Add to the buzz array if the number is evenly divisible by 5.
+  	- Add to the fizzbuzz array if the number is evenly divisible by 3 & 5.
+*/
+
+// Complete Exercise 12 below...
+
+var fizz = []
+var buzz = []
+var fizzbuzz = []
+nums.forEach(function(item){
+    if (item % 3 === 0) {
+    fizz.push(item);
+    } else if (item % 5 === 0) {
+      buzz.push(item)
+    } else if (item % 5 === 0 && item % 3 === 0) {
+      fizzbuzz.push(item)
+    }
+});
+console.log('Exercise 12 Results:');
+console.log(fizz)
+console.log(buzz)
+console.log(fizzbuzz)
+
+
+
+
+/*
+Exercise 13:
+  - Given the below numArrays array of arrays (two-dimensional array), assign the last nested array to a variable named numList.
+  - Assume you don't know how many nested arrays numArrays contains.
+*/
+
+const numArrays = [
+	[100, 5, 23],
+	[15, 21, 72, 9],
+	[45, 66],
+	[7, 81, 90]
+];
+
+
+
+// Complete Exercise 13 below...
+
+
+
+console.log('Exercise 13 Result:')
+var numList = []
+numList.push(numArrays.slice(-1)[0])
+console.log(numList)
+
+
+/*
+Exercise 14:
+  - Given the above numArrays array, access the number 66 and assign to a variable named num.
+*/
+
+// Complete Exercise 14 below...
+
+var num = numArrays[2][1]
+console.log('Exercise 14 Result:\n', num);
+
+
+/*
+Exercise 15:
+  - Given the above numArrays array, use nested forEach methods to sum up all the numbers contained within numArrays and assign to a variable named total.
+  - Hint: Be sure to declare and initialize the total variable before the iterations.
+*/
+
+// Complete Exercise 15 below...
+
+
+console.log('Exercise 15 Result:');
+ function totalItems(array) {
+
+  var total = 0;
+  array.forEach((item) => {
+    if(Array.isArray(item)) {
+     total += totalItems(item);
+    } else {
+    total += item;
+    }
+  })
+  return total;
+}
+console.log(totalItems(numArrays))
+
+//didn't know how to do #15 without writing a function. used this code as example: https://stackoverflow.com/questions/48791568/recursion-sum-nested-array
```

Comment:
what you did is right, here! not sure what you mean by 'without writing a function'?